Resources/Prototypes/_Backmen/Entities/Clothing/OuterClothing/coat.yml

```
@@ -175,9 +175,13 @@
         Slash: 0.7
         Piercing: 0.5
         ArmorPiercing: 0.85
+        Stun: 0.6
         Heat: 0.7
         Caustic: 0.75
         Stun: 0.4
```

YAMLlint (1.35.1)

[error] duplication of key "Stun" in mapping

(key-duplicates)